spec/mumukit/sections_spec.rb

```diff
@@ -12,6 +12,8 @@
                                                     'baz' => 'foobar' }
   it { expect(s.transform('foo' => 'bar',
                           'foobar' => 'foo /*<baz#*/lalala/*#baz>*/ ignored /*<bar#*/lelele/*#bar>*/')).to eq 'foo' => 'bar',
-                                                                                                              'baz' => 'lalala',
-                                                                                                              'bar' => 'lelele' }
+                                                                                                              'foobar' => {
+                                                                                                                  'baz' => 'lalala',
```

Comment:
The problem is that we are breaking backward compatibilty with exercises that perform interpolations, since now if you interpolate `baz`, nothing will be rendered, and I think there may be wollok exercises that rely on this. 

Nevertherless, the problem is that we should cut a new major semantic version. Perhaps we could just augment the original behaviour by: 

* putting the sections at the root level    
* and also, keeping the splited content as a hash.  

```ruby
{ 
  'foo' => 'bar',
  'baz' => 'lalala',
  'bar' => 'lelele', 
  'foobar' => {
     'baz' => 'lalala',
     'bar' => 'lelele'
   }
}
```